src/content/wiki/4.http-status-codes.md

```diff
@@ -0,0 +1,90 @@
+---
+title: All about HTTP Status Codes
```

Comment:
Maybe we want to have a title something like "Understanding different HTTP status code"